packages/react-components/react-message-bar/library/src/components/MessageBarGroup/useMessageBarGroupStyles.styles.ts

```diff
@@ -7,6 +7,7 @@ export const messageBarGroupClassNames: SlotClassNames<MessageBarGroupSlots> = {
   root: 'fui-MessageBarGroup',
 };
 
+// TODO: delete this obsolete code to save bytes?
```

Comment:
Yes, these styles should be deleted.

Reply:
> Yes, these styles should be deleted.

Done in https://github.com/microsoft/fluentui/pull/33465/commits/ee1613c0bb662ad26468f3643fdd740ebcd0bc69